src/api/artifact/abstractor/blob/fetcher.go

```diff
@@ -54,7 +59,7 @@ type fetcher struct{}
 // TODO re-implement it based on OCI registry driver
 func (f *fetcher) FetchManifest(repository, digest string) (string, []byte, error) {
```

Comment:
From the signature of this method, is it possible to return the Manifest object instead of bytes?

Reply:
There are three concrete Manifest objects at least: manifest version 1, manifest version2 and manifest list. The caller unmarshal the bytes into their own manifest object

Reply:
Not sure